corehq/apps/locations/static/locations/js/location_tree.async.js

```diff
@@ -250,7 +258,10 @@ hqDefine('locations/js/location_tree', function() {
         this.load_children_async = function(callback) {
             this.children_status('loading');
             api_get_children(this.uuid(), data.show_inactive, data.load_locs_url, function(resp) {
-                loc.set_children(resp);
+                loc.set_children(resp, {
+                    loc_edit_url: data.loc_edit_url,
```

Comment:
Minor: does `loc_edit_url` actually vary? Could you not pass it around as a parameter and instead just call `reverse("edit_location", loc_id)` wherever it's used? I wonder if it's originally a function just because it would have been ugly to have `'{% url "edit_location" domain "-locid-" %}'.replace('-locid-', loc_id);` repeatedly in the js.

Reply:
The only place it's called is in the html. I can probably just move the function to this file tho